pywwt/layers.py

```diff
@@ -0,0 +1,125 @@
+import re
+
+class ImageryLayers():
+    """
+    A supplemental class that standardizes layer names, sorts them by 
+    bandpass, and makes it possible to tab complete them.
+    """
+
+    def __init__(self, layer_list):
+        self._layers = {}
+        self._spectrum = ['gamma', 'x', 'uv', 'visible',
+                          'ir', 'micro', 'radio', 'other']
+        self.integers = ['zero', 'one', 'two', 'three', 'four',
+                         'five', 'six', 'seven', 'eight', 'nine']
+        for band in self._spectrum:
+            self._layers[band] = {}
+        
+        self._list2dict(layer_list)
+
+    def _list2dict(self, og_list):
+        # Helps turn the list of layer names used to initialize the class
+        # (og_list) into a dict.
+        for layer in og_list:
+            ind = 0
+
+            if re.search(r'(?i)gamma',layer) is not None:
+                self._add2dict(self._layers, layer, self._spectrum[ind])
+                continue # automatically advance to next iteration
+
+            ind += 1
+            if re.search(r'(?i)x(-|\s)?ray',layer) is not None:
+                self._add2dict(self._layers, layer, self._spectrum[ind])
```

Comment:
Just to check, why not just put ``'x'`` instead of ``self._spectrum[ind]`` here? I think it would be more readable if you just hard-code the bandpasses instead of relying on ``ind``.

Reply:
I was prioritizing portability over readability, but I can hard-code the bandpass names in the next commit.